holidayextras/holidayextras.github.com
# PROJECT LOUNGE - Adding a team page

## What does this PR do?
- Creates a team page for everyone at Holiday Extras who has written a blog post
- Uses a Gravatar for the persons photo, has a HX themed fallback if no Gravatar exists.
- Adds the users photo to their blog post

## How to test
- Checkout branch
- Run `jekyll serve` this should boot up the blog locally (if not try `bundle exec jekyll s`)
- Look at the team page and decided if it looks ok to release LIVE to the world
- Check a few blog posts for photos

## Review
### Quality checklist (for PR author to complete BEFORE code review):

#### IP or Developer Review:

#### Software Engineer, Software Architect or Developer review:
